packages/jimp/README.md

```diff
@@ -80,43 +80,65 @@ If you're using webpack you can set `process.browser` to true and your build of
+    // Do stuff with the image.
   })
   .catch(err => {
-    // handle an exception
+    // Handle an exception.
   });
-
-Jimp.read(lenna.buffer)
+  
+Jimp.read('http://www.example.com/path/to/lenna.jpg')
   .then(image => {
-    // do stuff with the image
+    // Do stuff with the image.
   })
   .catch(err => {
-    // handle an exception
+    // Handle an exception.
+  });
+  
+Jimp.read(jimpInstance)
+  .then(image => {
+    // Do stuff with the image.
+  })
+  .catch(err => {
+    // Handle an exception.
   });
 
-Jimp.read('http://www.example.com/path/to/lenna.jpg')
+Jimp.read(buffer)
   .then(image => {
-    // do stuff with the image
+    // Do stuff with the image.
   })
   .catch(err => {
-    // handle an exception
+    // Handle an exception.
   });
+```
+
+In some cases, you need to pass additional parameters alongside with an image's URL. You can pass an object to the `Jimp.read` method:
 
-Jimp.read({ url: 'http://www.example.com/path/to/lenna.jpg', headers: { ... } })
+```js
+Jimp.read({
+  url: 'http://www.example.com/path/to/lenna.jpg', // Required!
+  headers: {},
+  ...
+})
   .then(image => {
-    // do stuff with the image
+    // Do stuff with the image.
   })
   .catch(err => {
-    // handle an exception
+    // Handle an exception.
   });
 ```
 
-The conveniance method `Jimp.create` also exists. It is just a wrapper around `Jimp.read`.
+Jimp uses [phin](https://github.com/ethanent/phin) as Node.js HTTP client. Phin uses [`http.request(options[, callback])`](https://nodejs.org/dist/latest-v8.x/docs/api/http.html#http_http_request_options_callback) or [`https.request(options[, callback])`](https://nodejs.org/dist/latest-v8.x/docs/api/https.html#https_https_request_options_callback) methods (depends on a protocol) for making HTTP requests. Phin parses a passed `url` with the `url.parse(...)` method and pass it with all the other parameters as an `options` to the [`http.request(options[, callback])`](https://nodejs.org/dist/latest-v8.x/docs/api/http.html#http_http_request_options_callback) or [`https.request(options[, callback])`](https://nodejs.org/dist/latest-v8.x/docs/api/https.html#https_https_request_options_callback) methods.
+
+Briefly speaking, you can pass any options from [`http.request(options[, callback])`](https://nodejs.org/dist/latest-v8.x/docs/api/http.html#http_http_request_options_callback), [`https.request(options[, callback])`](https://nodejs.org/dist/latest-v8.x/docs/api/https.html#https_https_request_options_callback) or even [`tls.connect(options[, callback])`](https://nodejs.org/dist/latest-v8.x/docs/api/tls.html#tls_tls_connect_options_callback).
+
+Phin will care for a passed `url` (i.e. parse it) and pass a parsed `url` with any options you want. This behavior can be very useful when you need to pass some additional `headers`. Also, you can pass `rejectUnauthorized: false` if you don't require an SSL certificate to be valid (it helps to prevent `unable to verify the first certificate` error). 
```

Comment:
> Phin parses the URL and combines it with any options you want. This behavior can be very useful when you need to pass some additional headers. Also, you can pass `rejectUnauthorized: false` if you don't require an SSL certificate to be valid (it helps to prevent unable to verify the first certificate error).

Reply:
Should I remove "Briefly speaking...". I left if for now.